src/PlusDataCollection/vtkPlusDeviceFactory.cxx

```diff
@@ -415,6 +419,10 @@ vtkPlusDeviceFactory::vtkPlusDeviceFactory()
   RegisterDevice("AzureKinect", "vtkPlusAzureKinect", (PointerToDevice)& vtkPlusAzureKinect::New);
 #endif
 
+#ifdef PLUS_USE_REVOPOINTSURFACE
+  RegisterDevice("RevopointSurface", "vtkPlusRevopointSurface", (PointerToDevice)& vtkPlusRevopointSurface::New);

```

Comment:
Rename the class/device to indicate that it works with other Revopoint cameras. Maybe "vtkPlusRevopoint3DCamera".